src/views/PaymentNetworkFees.vue

```diff
@@ -0,0 +1,202 @@
+<script setup lang="ts">
+import snapshot from '@snapshot-labs/snapshot.js';
+
+const {
+  DEFAULT_CURRENCY,
+  DEFAULT_PLAN,
+  CURRENCIES,
+  PLANS,
+  BASE_PRICE,
+  BASE_UNIT,
+  BASE_CURRENCY,
+  fxRates,
+  transfer,
+  paymentTx,
+  loading,
+  fxLoadStatus
+} = usePayment(import.meta.env.VITE_DEFAULT_NETWORK);
+const { web3Account } = useWeb3();
+const modalPostPaymentOpen = ref(false);
+
+const data = reactive({
+  tos: false,
+  contract_signed: false,
+  currency: DEFAULT_CURRENCY,
+  plan: DEFAULT_PLAN
+});
+
+const amount = computed(() => {
+  return computePrice(data.currency, PLANS[data.plan].unit);
+});
+
+function setData(key: string, value: string | boolean) {
+  data[key] = value;
+}
+
+function pay() {
+  transfer(amount.value, data.currency);
+}
+
+function computePrice(
+  currencyId: string | null,
+  unit: number,
+  discount = 0
+): number {
+  const fxRate = currencyId ? fxRates[currencyId] : 1;
+  let price = (BASE_PRICE / fxRate) * unit;
+
+  if (discount > 0) {
+    price = price - (price / 100) * discount;
+  }
+
+  return price;
+}
+
+function computePlanFiatPrice(plan, unit?: number): number {
+  return computePrice(null, unit || plan.unit, plan.discount);
+}
+
+function formatFiatCurrency(amount: number): string {
+  return `${BASE_CURRENCY.name} ${
+    BASE_CURRENCY.symbol
+  }${amount.toLocaleString()}`;
+}
+
+function formatCryptoCurrency(amount: number, currency): string {
+  return `~${currency.code} ${amount.toLocaleString()}`;
+}
+
+watch(paymentTx, () => {
+  modalPostPaymentOpen.value = !!paymentTx.value;
+});
+</script>
+
+<template>
+  <TheLayout class="max-w-lg px-4 md:px-0">
```

Comment:
All the text inside the package and currency lists have only gray `text-skin-text` color. At least some of it should be `text-skin-heading`, could be only when it's selected.
<img width="544" alt="image" src="https://github.com/snapshot-labs/snapshot/assets/51686767/6fec61e4-1747-45ea-8110-bd1d87ebc700">